### ISSUE TITLE
Just updated - getting this error

### ISSUE THREAD

**Issue by realrover** [Mikadily/poloniexlendingbot#216]
```
Welcome to Poloniex Lending Bot
Started WebServer, lendingbot status available at http://localhost:8000/lendingbot.html
2017-02-06 11:53:27 Error: list index out of range
Traceback (most recent call last):
  File "C:\Utils\LendingBot\lendingbot.py", line 71, in <module>
    Lending.lend_all()
  File "C:\Utils\LendingBot\modules\Lending.py", line 160, in lend_all
    usable_currencies += lend_cur(cur, total_lended, lending_balances)
  File "C:\Utils\LendingBot\modules\Lending.py", line 268, in lend_cur
    cur_min_daily_rate = get_min_daily_rate(active_cur)
  File "C:\Utils\LendingBot\modules\Lending.py", line 179, in get_min_daily_rate
    recommended_min = Analysis.get_rate_suggestion(cur)
  File "C:\Utils\LendingBot\modules\MarketAnalysis.py", line 122, in get_rate_suggestion
    result = rates[index]
IndexError: list index out of range

Unhandled error, please open a Github issue so we can fix it!
```

**Comment by Evanito** [comment]
This is due to it attempting to find a suggestion from a blank list. 
A fix will be pushed soon, for now just wait for it to populate the lists in `market_data` a bit then it should run fine. Can you tell me if that solves it for you?

**Comment by realrover** [comment]
No, waiting doesn’t help. I run lendingbot.py from Windows command line. This error is generated every 1 minute until I close the window.

**Comment by Evanito** [comment]
What do the .csv files in your `market_data` folder look like? Are they empty/nonexistent?

**Comment by realrover** [comment]
No, look below.

 



 

_____________________

**Comment by Evanito** [comment]
I do not see anything below, is the problem resolved?

**Comment by realrover** [comment]
Sorry, I've inserted picture into the email, but it wasn't displayed here for some reason. Here it is again:

![image](https://cloud.githubusercontent.com/assets/4368457/22770450/99a71ce0-ee5e-11e6-9964-1b8b7b026334.png)

**Comment by Evanito** [comment]
Okay, please update to the most recent version which includes some fixes from today and see if your problem is fixed.

**Comment by realrover** [comment]
Here is what I get:
-------------------------------------------------------------------------
C:\Utils\LendingBot>lendingbot.py
WARN: Module Numpy not found, using manual percentile method instead. It is recommended to install Numpy. Error:
Importing the multiarray numpy extension module failed.  Most
likely you are trying to import a failed build of numpy.
If you're working with a numpy git repo, try `git clean -xdf` (removes all
files not under version control).  Otherwise reinstall numpy.

Welcome to Poloniex Lending Bot
Started WebServer, lendingbot status available at http://192.168.0.115:8000/lendingbot.html, http://127.0.0.1:8000/lendingbot.html
WARN: Exception found when analysing markets, if this happens for more than a couple minutes please make a Github issue so we can fix it. Ot
herwise, you can safely ignore it. Error: list index out of range
WARN: Exception found when analysing markets, if this happens for more than a couple minutes please make a Github issue so we can fix it. Ot
herwise, you can safely ignore it. Error: list index out of range
endDate reached. Bot can no longer lend.
Exiting...

**Comment by Evanito** [comment]
Thanks for putting up with the questions, found the bug and it has been squashed with extreme prejudice.
Update and you won't find this error anymore.
